cli/src/main/java/com/liferay/blade/cli/command/ServerRunCommand.java

```diff
@@ -117,7 +117,6 @@ else if (wildfly) {
 					processBuilderEnvironment.put("JAVA_OPTS", opts.toString());
 				}
 				else if (wildfly) {
-					commands.add("--debug");
```

Comment:
This is a fix for running wildfly with debug, this flag must be removed so that our custom JAVA_OPTS are used, or else wildfly doesn't start in debug mode.